srcalc/src/main/java/gov/va/med/srcalc/service/DefaultAdminService.java

```diff
@@ -104,7 +104,47 @@ public void saveVariable(final AbstractVariable variable)
     @Transactional(readOnly = true)
     public ImmutableCollection<RiskModel> getAllRiskModels()
     {
-        fLogger.debug("Getting all RiskModels.");
+//        fLogger.debug("Getting all RiskModels.");
         return fRiskModelDao.getAllRiskModels();
     }
+    
+    @Override
+    @Transactional(readOnly = true)
+    public RiskModel getRiskModelForId(final int modelId)
+    {
+//        fLogger.debug("Getting RiskModel {}.", modelId);
+        
+        final ImmutableCollection<RiskModel> models = getAllRiskModels();
```

Comment:
This actually does load quite a bit of information from the database. All risk models -> all terms -> all variables, etc. I think it's worth adding a DAO method to just load one risk model.